db/post_migrate/20221018091412_migrate_chat_channels.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateChatChannels < ActiveRecord::Migration[7.0]
+  def down
+  end
+
+  def up
+    DB.exec("UPDATE chat_channels SET type='CategoryChannel' WHERE chatable_type = 'Category'")
```

Comment:
We are dropping `chatable_type` in a future PR right?

Reply:
If possible in the future PR we should make sure any existing channel with chatable_type  `Tag`or `Site` should be dropped. I don't think we ever did this cleanup.

Reply:
They are still being used, `chatable_type`/`chatable_id` represent the link to the chatable object. `type` is only the type of the STI table inheriting from `ChatChannel` (d_m_channel, category_channel). Probably further improvements to be done in the future.

Reply:
Ah I see my bad, got turned around a bit from the existence of the other models :+1: